src/Connector/UserGroup/UserGroupConnector.php

```diff
@@ -0,0 +1,89 @@
+<?php
+namespace ExpandOnline\KlipfolioApi\Connector\UserGroup;
+use ExpandOnline\KlipfolioApi\Connector\BaseApiResourceConnector;
+use ExpandOnline\KlipfolioApi\Object\UserGroup\UserGroup;
+
+/**
+ * Class GroupConnector
+ * @package ExpandOnline\KlipfolioApi\Connector\UserGroup
+ */
+class UserGroupConnector extends BaseApiResourceConnector
+{
+    /**
+     * @var int
+     */
+    protected $userId;
+
+    /**
+     * @param string $option
+     * @return string
+     */
+    public function getEndpoint($option = '')
+    {
+        if (is_null($this->getUserId())) {
+            throw new \InvalidArgumentException('UserGroup must always have a User ID.');
+        }
+
+        return $this->formatEndpoint('users' . '/' . $this->getUserId() . '/' . 'groups'
```

Comment:
Als het UserGroup connector heet zou ik me bedenken dat setId de User is en er dan een setGroupId is